inc/modules/export/epub/templates/epub201/opf.php

```diff
@@ -45,6 +47,17 @@
 		echo '<dc:creator opf:role="aut"';
 		if ( ! empty( $meta['pb_author_file_as'] ) ) {
 			echo ' opf:file-as="' . $meta['pb_author_file_as'] . '"';
+		} else {
+			$nameparser = new Parser();
+			try	{
+				$author = $nameparser->parse( $meta['pb_author'] );
+				$author_file_as = $author->getLastName() . ', ' . $author->getFirstName();
+			} catch ( Exception $e ) {
```

Comment:
Should be `\Exception`. Optionally, my preference would be to catch `\HumanNameParser\Exception\NameParsingException` instead of searching the string for the error.